data/patientsTasks.csv

```diff
@@ -1,15 +1,6 @@
 PID,TID,DONE,RECURRENCE,DEADLINE,STARTTIME,ENDTIME,TASKTYPE,uuid
-1,2,false,false,09/11/2019 1400,,,S,1
-1,3,false,false,09/01/2018 1200,,,S,2
-1,3,false,false,04/04/2019 1400,,,S,3
-1,3,false,false,09/08/2020 1200,,,S,4
-1,3,false,false,09/05/2013 0900,,,S,5
-1,3,false,false,,16/09/2019 1400,12/08/2020 1200,E,6
-1,3,false,false,09/11/2019 1400,,,S,7
-2,7,false,false,,09/01/2019 1200,08/11/2019 1000,E,9
-2,7,false,false,09/05/2013 0150,,,S,10
-2,7,false,false,,09/11/2018 1400,08/08/2019 1200,E,11
-2,7,false,false,,08/08/2018 1400,09/09/2019 1200,E,12
-2,7,false,false,,09/08/2019 1200,09/12/2019 1200,E,13
-3,8,false,false,09/01/2022 1400,,,S,14
-3,5,false,false,,05/11/2019 1200,05/04/2022 1200,E,15
+2,7,false,false,,09/01/2019 1200,08/11/2019 1000,E,1
+2,7,false,false,09/05/2013 0150,,,S,2
+2,7,false,false,,09/11/2018 1400,08/08/2019 1200,E,3
+2,7,false,false,,08/08/2018 1400,09/09/2019 1200,E,4
+2,7,false,false,,09/08/2019 1200,09/12/2019 1200,E,5

```

Comment:
You should not commit csv. Or else everyone will need to resolve conflicts for this.

Reply:
noted